dotCMS/src/main/java/com/dotmarketing/portlets/templates/business/TemplateAPI.java

```diff
@@ -220,7 +219,7 @@ Template copy(Template sourceTemplate, Host destinationHost, boolean forceOverwr
 	 * @throws Exception
 	 */
 	@Deprecated
-	public boolean delete(Template template, User user, boolean respectFrontendRoles) throws DotSecurityException, Exception;
+	boolean delete(Template template, User user, boolean respectFrontendRoles) throws DotSecurityException, Exception;
```

Comment:
Codacy found an issue: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=6380036)